.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,22 +1,48 @@
 ---
-name: Bug Report
-about: Reporting issues with the VA API Platform
-
+name: 🔴 Bug Report
+about: For reporting issues with the VA API Platform
 ---
 
-## APIs Affected
+Important note: As this is a public forum, please do not include your VA API OAuth Client ID, VA API OAuth Client Secret, VA API key, or any Personally Identifiable Information (PII) or Protected Health Information (PHI) in your report.
+
+### Troubleshooting note
+* For test apps, please make sure you are using your registered URL, not the URL provided in the example.
+
+
+## Email address
+
+<!-- Please include your email address. -->
+
+
+## APIs affected
+
+<!-- Please list affected APIs (if applicable). -->
 
-<!-- Please list affected APIs -->
 
-## Brief Summary 
+## What I expected to happen
 
-<!-- One or two sentence summary of the issue. --> 
+<!-- What did you expect to happen? --> 
```

Comment:
Not sure this is needed, as it's already stated in the title. More important to give them specific direction on what information we need here (e.g. create a step by step summary of what the expected behavior was).

Reply:
👍 Modified to `<!-- Please provide a step by step summary of what the expected behavior was. -->` with a 1. 2. 3. to encourage a specific ordered list.